content/en/apps/features/targets/_index.md

```diff
@@ -6,40 +6,53 @@ description: >
 
-Counts without a goal display a simple black number count. Counts with a goal display the value of the goal on the right side and a colored count (green if the count is above the goal, or red if the count is below the goal).
+A count without a goal displays a simple black number count. A count with a goal displays the value of the goal on the right side and a colored count in the center indicating progress towards achieving the goal. Progress is displayed in green if the count is equal to or above the goal, or in red if the count is below the goal.
 
 ### Percent Widgets
 
-Percentage widgets provide insight into how much data matches a specific criteria against data that does not. This is calculated based on a true / false statement. This is often configured to represent accomplishment. For example, newborns should be visited within the first three days of life (“true”) can be displayed next to newborns that were not visited within the first three days (“false”).
+{{< figure src="mobile-percent.png" link="mobile-percent.png" class="right col-6 col-lg-3" >}}
 
-Next to the percentage, the count of reports used in the calculation are shown (e.g. 16 of 20 [newborn visits] on-time). CHWs have found this helpful in interpreting this information. 
+Percent widgets display a ratio of the number complete versus the total number possible for a given target. They provide insight into how much data of a particular report matches a specific criteria against data that does not match that same criteria. This is calculated based on a true / false statement. For example, newborns should be delivered in a facility (“true”) can be displayed next to newborns that were not delivered in a facility (“false”).
 
-An optional goal can be set, such as “70% of newborns should be visited within the first three days.” Conditional styling can be configured to show green if a goal has been met and red if the goal has not been met. 
+An optional goal can be set, such as “100% of patients with a fever should be given a malaria Rapid Diagnostic Test (mRDT),” to visualize progress towards achieving a target. Widget styling is configured to show green if the goal has been met and red if the goal has not been met. Next to the percent with a goal, the count of reports used in the calculation are shown (e.g. “16 of 20 with mRDT”). CHWs have found this helpful in interpreting target information.
+
+## Supervisor View
+
+In version 3.9, a Supervisor view was introduced to provide actionable information for CHW coaching and performance management. Targets can be configured to be aggregated and synced to allow Supervisors to view the progress of their health workers towards their goals. Within a Supervisors hierarchy branch, they can view:
+
+{{< figure src="mobile-supervisor.png" link="mobile-supervisor.png" class="right col-4 col-lg-6" >}}
```

Comment:
![image](https://user-images.githubusercontent.com/789512/84706659-5b64ab00-af2b-11ea-86bb-1e8592dd7e0d.png)

Just checking that two pages are correct, and that the item with 1 in 3 leads to a page showing it as 2 of 3.